d/dustjs-linkedin/dustjs_ubi_8.3.sh

```diff
@@ -0,0 +1,56 @@
+#----------------------------------------------------------------------------
+#
+# Package         : dustjs-linkedin
+# Version         : v3.0.0
+# Source repo     : https://github.com/linkedin/dustjs.git
+# Tested on       : ubi:8.3
+# Script License  : Apache License, Version 2 or later
+# Maintainer      : srividya chittiboina <Srividya.Chittiboina@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+#
+#
+# ----------------------------------------------------------------------------
+
+set -e
+
+WORK_DIR=`pwd`
+
+REPO_URL=https://github.com/linkedin/dustjs.git
+
+PACKAGE_NAME=dustjs
+
+VERSION=${1:-v3.0.0}
+
+dnf install git wget nodejs -y
```

Comment:
is there a specific reason for using nodejs 10? ideally we should be using v12 which can be installed using nvm or "dnf module enable nodejs:12; dnf install nodejs"

Reply:
yes @seth-priya version 12 also works fine I will update the script and commit